src/common/index.ts

```diff
@@ -260,37 +260,37 @@ export type MenuThemeDescription = {
    * the menu away from the screen edges when it's opened too close to them. Default is
    * 150px.
    */
-  readonly maxMenuRadius: number;
+  maxMenuRadius: number;
 
   /** The width of the text wrap in the center of the menu in pixels. Default is 90px. */
-  readonly centerTextWrapWidth: number;
+  centerTextWrapWidth: number;
 
   /**
    * If this is true, children of a menu item will be drawn below the parent. Otherwise
    * they will be drawn above. Default is true.
    */
-  readonly drawChildrenBelow: boolean;
+  drawChildrenBelow: boolean;
 
   /**
    * If this is set to true, the center text of the menu will be drawn. This is the text
    * that is displayed in the center of the menu when it is opened. Default is true.
    */
-  readonly drawCenterText: boolean;
+  drawCenterText: boolean;
 
   /**
    * If this is set to true, a full-screen div will be drawn below the menu with the CSS
    * class "selection-wedges". If any menu item is hovered, it will also receive the class
    * "hovered" and the "--start-angle" and "--end-angle" CSS properties will indicate
    * where the selected child is. Default is false.
    */
-  readonly drawSelectionWedges: boolean;
+  drawSelectionWedges: boolean;
 
   /**
    * If this is set to true, a full-screen div will be drawn below the menu with the CSS
    * class "wedge-separators". It will contain a div for each separator line between
    * adjacent wedges. They will have the "separator" class. Default is false.
    */
-  readonly drawWedgeSeparators: boolean;
+  drawWedgeSeparators: boolean;
```

Comment:
These changes can be reverted since you removed those assignment usages in https://github.com/kando-menu/kando/commit/2077b5500dbd00eb1e3beaa28da0986a0227cd1a.